docs/sources/installation/windows.md

```diff
@@ -150,6 +150,12 @@ You can do this with
 - then click: "Save Private Key".
 - Then use the saved file to login with PuTTY using `docker@127.0.0.1:2022`.
 
+## Uninstall Boot2Docker
+
+Boot2Docker can be uninstalled using the normal Windows process for uninstalling
+a program. This will not delete the `docker-install.exe`, but you can simply 
+delete the file at any time.
```

Comment:
You can uninstall Boot2Docker using Window's standard process for removing programs.  This process does not remove the `docker-install.exe` file. You must delete that file yourself.

(The any time is kind of assumed and a single sentence is stronger using the reflexive pronoun.)